mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(prices) {
+  return 0.2 * prices;
```

Comment:
It seems that the test has failed. Would you be able to fix this?

Reply:
Has it ? It passed on my machine . I will apply toFixed(2)